src/components/info/InfoProvider.tsx

```diff
@@ -9,7 +9,7 @@ import {
 export interface IInfoContext {
     supportedTemplates: {
         manifests: string[];
-        metadata: string[];
+        assays: string[];
```

Comment:
This change is kind of tangential to the purpose of this PR - it makes the `InfoContext` more consistent with the `/info/assays` endpoint.